src/nodes/Networking/Inline.js

```diff
@@ -185,7 +185,7 @@ x3dom.registerNodeType(
                 //Return contentType if defined & valid
                 if (this._vf.contentType != "" && this.isValidContentType( this._vf.contentType ) )
                 {
-                    return contentType;
+                    return this.ContentType[this._vf.contentType];
```

Comment:
It should be: return this._vf.contentType;

I already commit the change to master